src/SDK/Client/Platform/methods/documents/broadcast.ts

```diff
@@ -1,9 +1,17 @@
+import {Platform} from "../../Platform";
+import broadcastStateTransition from '../../broadcastStateTransition';
+
 /**
  * Broadcast document onto the platform
  *
  * @param {Platform} this - bound instance class
- * @param document - document
+ * @param documents - documents
  * @param identity - identity
  */
-import broadcast from "../broadcastRecords";
-export default broadcast;
+export default async function broadcast(this: Platform, documents: { create: any[], replace: any[], delete: any[]}, identity: any): Promise<any> {
+    const { dpp } = this;
+
+    const documentsBatchTransition = dpp.documents.createStateTransition(documents);
+
+    await broadcastStateTransition(this, documentsBatchTransition, identity);
```

Comment:
In docs, you return docs. Does it make sense to return anything from here?

Reply:
I want to return documents to not be void, but yeah, may be not best solution ?

Reply:
I would return void from broadcast methods for now and later figure out what is most convenient thing can be for users.

Reply:
I did the opposite by just returning documents.